src/pages/CompanyTrainingOverview/CTOPage.jsx

```diff
@@ -0,0 +1,97 @@
+// Import necessary components from Material-UI library
+import Button from "@mui/material/Button";
+import { List, ListItem, ListItemText } from "@mui/material";
+// Import custom PageLayout component
+import PageLayout from "../../components/PageLayout";
+
+// CTOPage component definition
+function CTOPage() {
+  // Return JSX markup for the CTOPage component
+  return (
+    // Use the custom PageLayout component with specific title and viewport settings
+    <PageLayout title="CTO" viewportPage>
+
+      {/* Container for the page content */}
+      <div
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100vh",
+          padding: "20px", // Adjust as needed
+        }}
+      >
```

Comment:
sx is only allowed to style MUI-Components

Reply:
Adapted div to Box